modular.html

```diff
@@ -275,16 +275,448 @@ <h3>Gradle</h3>
+    and the build system will download the required modules, including the native libraries 
+    for your platform.
+</p>
+<p>
+    A simple Maven example can be found in this  
+    <a class="samples" href="/CommandLine/Non-modular/Maven" target="_blank">project</a>.
+    The <a class="samples" href="/CommandLine/Non-modular/Maven/hellofx/pom.xml" target="_blank">pom.xml</a> 
+    file shows how to create a runnable fat jar for this <kbd>hellofx</kbd> project with the Maven shade plugin.
+</p>
+
+<p>
+    Run the plugin to create the fat jar:
+</p>
+
+<pre><code>
+mvn compile package
+</code></pre>
+
+<p>
+    Run the application using:
+</p>
+
+<ul class="nav nav-tabs">
+    <li class="nav-item">
+        <a class="nav-link active" href="#nix-maven" data-toggle="tab">Linux/Mac</a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="#win-maven" data-toggle="tab">Windows</a>
+    </li>
+</ul>
+<div class="tab-content">
+    <div class="tab-pane active" id="nix-maven">
+<pre class="no-border-radius"><code>
+java -jar shade/hellofx.jar
+</code></pre>
+    </div>
+    <div class="tab-pane" id="win-maven">
+<pre><code>
+java -jar shade\hellofx.jar
+</code></pre>
+    </div>
+</div>
+
+<h5>Cross-platform jar</h5>
+
+<p>
+    You can create a cross-platform fat jar by adding the dependencies from the 
+    three platforms that require native libraries to your pom file, in this sample 
+    only the ones from <kbd>javafx.graphics</kbd>:
+</p>
+
+<textarea rows="22" style="border:none;width:100%" readonly>
+    <dependencies>
+        ...
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-graphics</artifactId>
+            <version>11</version>
+            <classifier>win</classifier>
+        </dependency>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-graphics</artifactId>
+            <version>11</version>
+            <classifier>linux</classifier>
+        </dependency>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-graphics</artifactId>
+            <version>11</version>
+            <classifier>mac</classifier>
+        </dependency>
+    </dependencies>
+</textarea>
+
+<h4>Gradle</h4>
+
+<p>
+    If you develop your JavaFX applications using Gradle, you don't have to download
+    the JavaFX SDK. Just specify the modules and the versions you want in the <kbd>build.gradle</kbd> file,
+    and the build system will download the required modules, 
+    including the native libraries for your platform. However, for Gradle we need to 
+    find and specify the platform/OS as classifier.
+</p>
 
 <p>
-    If you have a non-modular JavaFX <span class="JFX_MAJOR">11</span> project and you can't use <kbd>jlink</kbd>, you can still do a <a href="index.html#fat-jar">fat-jar</a>.
+    A simple Gradle example can be found in this  
```

Comment:
This project still doesn't make use of the new Gradle JavaFX plugin. Is it by intention?

Reply:
When the PR was raised the plugin version didn't work for non-modular projects, so that will have to be updated later on in a follow-up PR for this and all the Gradle samples in the docs.